src/Components/ExperimentTwo/ExperimentTwo.js

```diff
@@ -1,5 +1,39 @@
-function ExperimentTwo() {
-  return <h1>Welcome to Experiment 2</h1>;
-}
+import { useState } from "react";
+import Button from "../ReUsableUIs/Button.js";
+import TrendingFilmsTable from "../ExperimentTwo/TrendingFilmsTable.js";
+
+const ExperimentTwo = (props) => {
+  const [movieButtonState, setMovieButtonState] = useState(null);
+
+  if (movieButtonState == null) {
+    return (
+      <>
+        <Button
+          textToDisplay="What are the Trending Movies these days"
+          onClickListener={() => setMovieButtonState(1)}
+        />
+
+        <Button textToDisplay="Take power and choose your MOVIE" />
+
+        <Button textToDisplay="Lets recommend you some movies" />
+
+        <Button
+          textToDisplay="Head to Main Arena"
+          onClickListener={props.isBackButtonClicked}
+        />
+      </>
+    );
+  } else {
+    return (
+      <>
+        {movieButtonState == 1 && (
```

Comment:
can this condition just be `movieButtonState && ...` ?

Reply:
either way, you should avoid `==` and use `===` instaead

Reply:
Oh yeah I should have made it clearer. Basically,  `movieButtonState==1` will be extended to 2,3 .. in the next features. Hence why I used this way. 
Thanks for mentioning about the `===`. Updated